catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/pipelines/PipelineResourceTest.java

```diff
@@ -522,22 +516,34 @@ void delete_nonEmptyPipeline_4xx() {
   /** Validate returned fields GET .../pipelines/{id}?fields="..." or GET .../pipelines/name/{fqn}?fields="..." */
   @Override
   public void validateGetWithDifferentFields(Pipeline pipeline, boolean byName) throws HttpResponseException {
-    // .../Pipelines?fields=owner
-    String fields = "owner";
+    String fields = "";
     pipeline =
         byName
             ? getPipelineByName(pipeline.getFullyQualifiedName(), fields, ADMIN_AUTH_HEADERS)
             : getPipeline(pipeline.getId(), fields, ADMIN_AUTH_HEADERS);
-    assertListNotNull(pipeline.getOwner(), pipeline.getService(), pipeline.getServiceType());
-    assertNull(pipeline.getTasks());
-
-    // .../Pipelines?fields=owner,service,tables
-    fields = "owner,tasks";
+    assertListNotNull(pipeline.getService(), pipeline.getServiceType());
+    assertListNull(
+        pipeline.getOwner(),
+        pipeline.getTasks(),
+        pipeline.getPipelineStatus(),
+        pipeline.getTags(),
+        pipeline.getFollowers(),
+        pipeline.getTags());
+
+    fields = "owner,tasks,pipelineStatus,followers,tags";
     pipeline =
         byName
             ? getPipelineByName(pipeline.getFullyQualifiedName(), fields, ADMIN_AUTH_HEADERS)
             : getPipeline(pipeline.getId(), fields, ADMIN_AUTH_HEADERS);
-    assertListNotNull(pipeline.getOwner(), pipeline.getService(), pipeline.getServiceType(), pipeline.getTasks());
+    assertListNotNull(pipeline.getService(), pipeline.getServiceType());
+    // Some fields are not set and hence are null - tested elsewhere
+    assertListNotNull(
+        pipeline.getOwner(),
+        /*pipeline.getTasks(),
```

Comment:
Remove commented code?

Reply:
That was to call out, we are not specifically testing those fields... :)